### ISSUE TITLE
Scrollbar hides copy-to-editor angle brackets on macOS on hover

### ISSUE THREAD

**Issue by jezdez** [getredash/redash#4919]
### Issue Summary

When scrolling the schema browser on macOS the copy-to-editor angle brackets are hidden by the system scrollbar until it fades away again. 

### Steps to Reproduce

1. Scroll in the schema browser and trigger the wider native system scrollbar on macOS
2. See the angle brackets to copy the table name to shortly be hidden

Any other info e.g. Why do you consider this to be a bug? What did you expect to happen instead?

![CleanShot 2020-05-25 at 22 02 53](https://user-images.githubusercontent.com/1610/82840687-cba38200-9ed3-11ea-845c-1b85adb494e7.gif)

### Technical details:

* Redash Version: ce6bc2d64aa1bff38c9d461ba4f5387627fa18c1
* Browser/OS: macOS/Firefox/Chrome
* How did you install Redash: Docker

**Comment by gabrieldutra** [comment]
I wouldn't classify this as a bug myself as it's part of the "automatic show scrollbar" macOS feature:

<img width="652" alt="Screen Shot 2020-05-25 at 17 19 36" src="https://user-images.githubusercontent.com/3356951/82841302-ee21a500-9eab-11ea-9a3d-71ab63d84b7d.png">

**Comment by jezdez** [comment]
Yeah that was my first thought as well when one of our users reported it and I wanted to let you know and see what you think. It's only a minor nuisance really :)

**Comment by arikfr** [comment]
Although I don't see the big issue here, let's add padding on the right to compensate for a potential scrollbar.
